scripts/runfabtests.sh

```diff
@@ -2,23 +2,46 @@
 
 trap cleanup_and_exit SIGINT
 
+#
+# Default behavior with no args will use sockets provider with loopback
+#
 declare BIN_PATH
-declare PROV
+declare PROV="sockets"
 declare TEST_TYPE="quick"
-declare SERVER
-declare CLIENT
+declare SERVER="127.0.0.1"
+declare SSH_SERVER
+declare CLIENT="127.0.0.1"
+declare SSH_CLIENT
 declare EXCLUDE
 declare GOOD_ADDR="192.168.10.1"
 declare -i VERBOSE=0
+declare XARGS_CMD
+
+declare OSNAME=`uname -s | tr '[A-Z]' '[a-z]'`
+
+if [ -z "$TMPDIR" ]; then
+    TMPDIR=/tmp/
+fi
 
 # base ssh,  "short" and "long" timeout variants:
 declare -r bssh="ssh -n -o StrictHostKeyChecking=no -o ConnectTimeout=2 -o BatchMode=yes"
-declare -r sssh="timeout 60s ${bssh}"
-declare -r lssh="timeout 90s ${bssh}"
+if [ "$OSNAME" == "linux" ]; then
+    declare -r sssh="timeout 60s ${bssh}"
+    declare -r lssh="timeout 90s ${bssh}"
+    XARGS_CMD="xargs -r"
```

Comment:
In that case, the piped command line fails, but we can ignore the resulting error.  Might make it harder to figure out if other parts of it failed, which probably is only going to be a problem when porting to new OS (e.g, Windows).